docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/melt.mdx

```diff
@@ -46,14 +46,13 @@ https://github.com/iotaledger/iota-sdk/blob/develop/sdk/examples/how_tos/native_
 
 1. Instantiate a [`Wallet`](https://docs.rs/iota-sdk/latest/iota_sdk/wallet/core/struct.Wallet.html), get Alice's `Account` which was
    [created in the first guide](../accounts-and-addresses/create-account.mdx) and [sync it](../accounts-and-addresses/check-balance.mdx).
-2. Prepare to melt `MELT_AMOUNT` native tokens transaction by calling the [`Account.prepareMeltNativeToken()`](../../references/nodejs/classes/Account.md#preparemeltnativetoken)
-   function anb send the prepared transaction using the [`Account.send()`](../../references/nodejs/classes/Account.md#send)
+2. Create a transaction to melt `MELT_AMOUNT` native tokens by calling the [`Account.meltNativeToken()`](../../references/nodejs/classes/Account.md#meltnativetoken) <!-- TODO Update ref link --> 
```

Comment:
```suggestion
2. Create a transaction to melt `MELT_AMOUNT` native tokens by calling the [`Account.meltNativeToken()`](../../references/nodejs/classes/Account.md#meltnativetoken) 
```

Reply:
We should leave them until I fix the refs